conans/test/integration/command_v2/list_recipe_revisions_test.py

```diff
@@ -0,0 +1,181 @@
+import re
+
+import pytest
+
+from conans.test.assets.genconanfile import GenConanfile
+from conans.test.utils.tools import TestClient, TestServer
+
+
+class TestListRecipeRevisionsBase:
+    @pytest.fixture(autouse=True)
+    def _setup(self):
+        self.users = {}
```

Comment:
Unused self.users member?

Reply:
Yes, it's a legacy variable. Good catch!